src/axom/mint/docs/sphinx/sections/mesh_representation.rst

```diff
@@ -0,0 +1,330 @@
+  lowest dimensional constituent entities of a mesh. The :ref:`Nodes` are
+  associated with the spatial coordinates of the mesh and are used in defining
+  the topology of the higher dimensional topological entities comprising the mesh,
+  such as, the :ref:`Cells`, :ref:`Faces`, etc., as discussed earlier.
+  In a sense, the :ref:`Nodes` provide the means to link the :ref:`Topology`
+  of the mesh to its constituent :ref:`Geometry` and thereby instantiate the mesh
+  in physical space.
+
+.. admonition:: Definition
+
+   A mesh node, :math:`\mathcal{n_i}`, is associated with a point,
+   :math:`p_i \in \mathbb{R}^d` and provides the means to:
+
+   #. Link the :ref:`Topology` of the mesh to its constituent :ref:`Geometry`
+
+   #. Support one or more *degrees of freedom*, evaluated at the given node
+      location.
+
+
+Notably, the nodes of a mesh are not necessarily just the *vertices* of the mesh.
+As discussed in the :ref:`PreliminaryConcepts` section, a mesh is a
+discretization of a PDE. Recall, the primary purpose of the mesh is to define
+the dicsrete locations, in both *space* and *time*, at which the
+*unknown variables* or *degrees of freedom* of the governing PDE are evaluated.
+Depending on the numerical scheme employed and the :ref:`CellTypes` used, the
+:ref:`Nodes` of a mesh may also be located at cell, face and edge centroids.
+For example, in the Finite Element Method (FEM), the nodes for the linear
+Lagrange Finite Elements, see :numref:`figs/linearCells`, are located at the
+cell *vertices*. However, for quadratic :ref:`CellTypes`,
+see :numref:`figs/q2cells`, the *Lagrange* :math:`P^2` finite element,
+for the quadrilateral and hexahedron (in 3D) cells, includes as :ref:`Nodes`,
+the cell, face and edge (in 3D) centroids in addition to the cell *vertices*.
+Other higher order finite elements may involve additional nodes for each edge
+and face as well as in the interior of the cell.
+
+.. _Connectivity:
+
+Connectivity
+""""""""""""
+
+The topological connections or *adjecencies* between the :ref:`Cells`,
+:ref:`Faces` and :ref:`Nodes` comprising the mesh, give rise to a hierarchical
+topological structure, depicted in :numref:`figs/topological_structure`, that is
+broadly referred to as :ref:`Connectivity` information. At the top level, a
+mesh consists of one or more :ref:`Cells`, which constitute the highest
+dimensional entity comprising the mesh. Each cell is bounded by zero or more
+:ref:`Faces`, each of which is bounded by one or more :ref:`Nodes`.
+
+.. _figs/topological_structure:
+.. figure:: ../figures/topological_structure.png
+  :align: center
+  :scale: 95%
+  :alt: Topological Structure
+
+  Hierarchical topological structure illustrating the *downward* and *upward*
+  topological connections of the constituent mesh entities supported in Mint.
+
+  The topological connections between the constituent entities of the mesh can be
```

Comment:
Un-indent here to fix runaway centering (since this should not be part of the figure caption)